.github/workflows/devbuild.yml

```diff
@@ -167,6 +168,7 @@ jobs:
           python3 -m pip -v install --upgrade setuptools
           python3 -m pip -v install --upgrade pip
           python3 -m pip -v install --upgrade numpy pytest flake8
+          python3 -m pip -v install --upgrade pyside6 shiboken6
```

Comment:
I developed and tested `-DMODMESH_USE_PYSIDE` on macos.  It is the only platform that I enable it on.  I will need ubuntu and windows devs to help the other platforms.

Reply:
I would like to help test Windows and Linux; however, the biggest problem now is neither Windows nor Linux could run apps in the viewer. Before we fix the viewer, this feature is not testable. @yungyuc

Reply:
Thanks for reporting the issue.  I will find time to take a look at Linux build.